spec/controllers/wikis_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe WikisController, type: :controller do
+
+  context 'not logged in' do
```

Comment:
as a general rule of thumb, contexts should always appear within `describe` blocks

```ruby
describe 'GET #show' do
  context 'when not logged in' do
    # ...test cases
  end

  context 'when logged in' do
        # ...test cases
  end
end
```